### PR TITLE
Deploy descheduler, upgrade Helm charts 

### PR DESCRIPTION
**Pull request by ronardcaktus**
### Issue
- https://app.clickup.com/t/8677fzttd

### Summary
- [x] descheduler chart install: v0.22.1
- [x] ingress-nginx chart upgrade: v4.0.19 → v4.4.2
- [x] cert-manager chart upgrade: v1.7.2 → v1.11.0
- [x] newrelic chart upgrade: v3.5.1 → v5.0.4
- [x] AWS NodeGroup AMI upgrade: 1.22.9-20220620 → 1.22.17-20230217

#### Deploy descheduler

```
❯ inv deploy.install
❯ inv deploy.playbook -n deploy-cluster.yml
❯ kubectl -n kube-system get cronjob
NAME          SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
descheduler   */2 * * * *   False     0        59s             6m11s
```

#### Upgrade ingress-nginx and cert-manager

```
❯ inv deploy.playbook -n deploy-cluster.yml
❯ kubectl -n ingress-nginx get pod -o wide 
NAME                                        READY   STATUS    RESTARTS   AGE     IP            NODE                                        NOMINATED NODE   READINESS GATES
ingress-nginx-controller-7b454df556-ds79n   1/1     Running   0          2m17s   10.0.15.220   ip-10-0-14-203.us-east-2.compute.internal   <none>           <none>
ingress-nginx-controller-7b454df556-z4hl7   1/1     Running   0          117s    10.0.10.241   ip-10-0-10-229.us-east-2.compute.internal   <none>           <none>

❯ helm -n cert-manager list
NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                   APP VERSION
cert-manager    cert-manager    4               2023-02-24 21:35:02.727236341 +0000 UTC deployed        cert-manager-v1.7.2     v1.7.2
```

#### Upgrade newrelic chart

```
❯ helm -n newrelic uninstall newrelic-bundle
release "newrelic-bundle" uninstalled
❯ inv production deploy.playbook -n deploy-hosting-services.yml
❯ helm -n newrelic list
NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                   APP VERSION
newrelic-bundle newrelic        1               2023-02-24 21:41:36.95967101 +0000 UTC  deployed        nri-bundle-5.0.4                   
```

New nodes running (approx 10 mins 4:50 - 5:00 PM):

Screenshot 2023-02-24 at 5 00 28 PM

### CODE REVIEW

File: deploy/requirements.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 - src: https://github.com/caktus/ansible-role-k8s-web-cluster
   name: caktus.k8s-web-cluster
-  version: v1.3.0
+  version: v1.4.0
 
 - src: https://github.com/caktus/ansible-role-k8s-hosting-services
   name: caktus.k8s-hosting-services
```
Comment (copelco) [inline deploy/requirements.yml:16]: Maybe update caktus.k8s-hosting-services to v0.9.0 and re-run the deploy-hosting-services.yaml playbook? It includes helpful defaults for New Relic

File: deploy/group_vars/all.yml [hunk 2]
```diff
@@ -77,7 +92,7 @@ k8s_iam_users: [copelco]
 # Pin ingress-nginx and cert-manager to current versions so future upgrades of this
 # role will not upgrade these charts without your intervention:
 # https://github.com/kubernetes/ingress-nginx/releases
-k8s_ingress_nginx_chart_version: "4.0.19"
+k8s_ingress_nginx_chart_version: "4.4.2"
```
Comment (copelco) [inline deploy/group_vars/all.yml:95]: Can you add the helm list output for ingress-nginx and cert-manager?

Comment (ronardcaktus) [reply deploy/group_vars/all.yml:95]: Done, it's in **Upgrade ingress-nginx and cert-manager** section.

File: deploy/requirements.yml (hunk 1 above)
Comment (ronardcaktus) [reply deploy/requirements.yml:16]: Done and done.

**Review by copelco — APPROVED** [review]

**Pull request merged**